subfun/BIDS_RFX.m

```diff
@@ -51,130 +54,130 @@ function BIDS_RFX(action, mmFunctionalSmoothing, mmConSmoothing, opt)
 % TASK NAME
 ExperimentName = opt.taskName;
 
-origdir = pwd;
+%origdir = pwd;
 
```

Comment:
origdir and WD are both in the script and they both point to the same location.
so I removed origdir , and kept WD for consistency among scripts.